docs/ignoring-files-folders-code.md

```diff
@@ -0,0 +1,186 @@
+| To include files or folders defined within a `.gitignore` for a particular scan. | Run Semgrep with the flag `--no-git-ignore`, which will override any definitions in `.semgrepignore` as well. |
+| To ignore files or folders for a particular rule | Edit the rule to set the `paths` key with one or more patterns. See [Rule syntax](../writing-rules/rule-syntax#paths).
+
+
+## Defining files and folders in `.semgrepignore`
+
+`.semgrepignore` syntax mirrors `.gitignore` syntax, with the following modifications:
+
+* "Include" patterns (lines starting with `!`) are unsupported.
+* "Character range" patterns (lines including a collection of characters inside brackets) are unsupported.
+* An `:include ...` directive is added, which allows another file to be included in the ignore pattern list; typically this included file would be the project `.gitignore`. No attempt at cycle detection is made.
+* Any line beginning with a colon, but not `:include`, will raise an error.
+* `\:` is added to escape leading colons.
+
+Unsupported patterns are silently removed from the pattern list (this is done so that `.gitignore` files may be included without raising errors). The removal will be logged.
+
+For a description of `.gitignore` syntax, see [.gitignore documentation](https://git-scm.com/docs/gitignore).
+
+
+## Defining files and folders in Semgrep App
+
+Another method for users to define ignore patterns is through a Project in Semgrep App. These patterns follow the same syntax as `.semgrepignore` in the preceding section.
+
+To define files and folders in Semgrep App:
+
+1. Sign into [Semgrep App](https://semgrep.dev/login?return_path=/manage/projects).
+2. From the Dashboard Sidebar, select **Projects > [Project name]**.
+3. Enter files and folders to ignore in the **Path Ignores** box.
+
+Including files and folders through this method is **additive**. When Semgrep App makes a scan, it will look for a `.semgrepignore` within the repository. If no `.semgrepignore` file is found, it will temporarily create one and add items from Semgrep App's Path Ignores. Adding items to the **Path Ignores** box will not override default Semgrep ignore patterns.
+
+
+## Ignoring code through nosemgrep
+
+To ignore blocks of code, define an **inline comment**, followed by a **space** ( ), followed by the word `nosemgrep` at either the **first line** or **the line preceding** the potential match. Semgrep will ignore all rule pattern matches. This functionality works across all supported languages.
+
+`nosemgrep` in Python:
+
+```python
+
+bad_func1()  # nosemgrep
+
+# nosemgrep
+bad_func2()
+
+```
+
+`nosemgrep` in JavaScript:
+
+```javascript
+
+bad_func1(); // nosemgrep
+
+bad_func2(   // nosemgrep
+    arg
+);
+
+```
+
+:::info
+
+The space ( ) before `nosemgrep` is required for Semgrep to detect this annotation.
```

Comment:
Same comment as the comment for line 110

Reply:
Updated in latest docs. Resolving both tickets.